src/components/Modals/AddMealModal.js

```diff
@@ -1,73 +0,0 @@
-import React, { Component } from "react";
-import { Button, Modal, ModalHeader, ModalBody, ModalFooter } from "reactstrap";
-import { AvForm } from "availity-reactstrap-validation";
-import { MealForm } from "../MealPlanner/Form";
```

Comment:
ok, what about this component?

Reply:
this is the file i just deleted

Reply:
check the latest line here. my comment related to it

Reply:
i'm sorry but, did you mean the others components that i updated?

Reply:
MealPlanner/Form component should be removed too.